installer/centos/anaconda_installer.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+# get install prefrix from cmd line
+if test "$#" -ne 1; then
+    echo "please specify an install prefix"
+fi
+
+# expand provided path
+if [[ ! "$1" =~ ^/ ]]
+    then
+        install_prefix="$PWD/$1"
+    else
+        install_prefix="$1"
+fi
+echo "Installing hnn, iv and nrn into $install_prefix"
+
+# check system dependencies.
+# TODO: What else should be checked for
```

Comment:
it worked for me without installing any system dependencies on CentOS

Reply:
Try again with a fresh base install. There are many dependencies not included by default.